portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,47 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.h1--container {
```

Comment:
Is this purposefully left empty? 
Also replace -- with single -

Reply:
@Lev-Stambler if you were trying to follow BEM syntax, `-` is for descendants and `--` is for a modifier, eg:

```
.component {...}
.component-title {...}
.component-title--large {...}
```

Reply:
Oh ok, I started looking into it, but never fully understood it.